include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
+public:
+	enum loglevel
+	{
+		logCRITICAL, logERROR, logWARNING, logDEBUG, logINFO,
+	};
+	Logger(std::string file_name, loglevel level) : full_buffer_size (100)
+	{
+		log_level_needed = level;
+		out.open(file_name, std::ofstream::out | std::ofstream::trunc);
+	}
+	~Logger()
+	{
+		out << full_buffer;
+		out.close();
+	}
+	const char* timenow()
+	{
+		auto timenow = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+		return  ctime(&timenow);
```

Comment:
calling this function and returning pointer to it's internal buffer might be dangerous.
try to save data to std::string and return it from the function instead. we might update this when will implement engine frame system